src/rapids_pre_commit_hooks/pyproject_license.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import random
+
+import tomlkit
+import tomlkit.exceptions
+
+from .lint import LintMain
+
+RAPIDS_LICENSE = "Apache 2.0"
```

Comment:
Nearly everything in RAPIDS is Apache 2, but anything `ucx` related uses [`BSD-3-Clause`](https://github.com/rapidsai/ucxx/blob/branch-0.38/LICENSE).

Reply:
I've added a set of acceptable licenses, defaulting to `Apache 2.0` if it doesn't match any of them.